pkg/convert/pot.go

```diff
@@ -73,46 +53,175 @@ func FromPot(b []byte) (model.Messages, error) {
 	}
 
 	messages := make([]model.Message, 0, len(po.Messages))
-	// model.Messages does not support plurals atm. But we plan to impl it under:
-	// https://github.com/orgs/translate-agent/projects/1?pane=issue&itemId=21251425
+
 	for _, node := range po.Messages {
-		if strings.HasSuffix(node.MsgId, "\\n") {
-			node.MsgId = strings.ReplaceAll(node.MsgId, "\\n", "\n")
+		message := model.Message{ID: node.MsgId, PluralID: node.MsgIdPlural}
+
+		switch {
+		case po.Header.PluralForms.NPlurals > pluralCountLimit:
+			return model.Messages{}, errors.New("plural forms with more than 2 forms are not implemented yet")
+		case po.Header.PluralForms.NPlurals > 1:
+			message.Message = convertToFormatMessage(node.MsgStr)
+		default:
+			message.Message = node.MsgStr[0]
 		}
 
-		if node.MsgIdPlural != "" {
-			continue
+		message.Description = strings.Join(node.ExtractedComment, "\n ")
+		message.Fuzzy = strings.Contains(node.Flag, "fuzzy")
+
+		messages = append(messages, message)
+	}
+
+	return model.Messages{
+		Language: po.Header.Language,
+		Messages: messages,
+	}, nil
+}
+
+func writeToPoTag(b *bytes.Buffer, tag poTag, str string) error {
+	var write func(*bytes.Buffer, poTag, string) error
+
+	switch tag { //nolint:exhaustive
+	case MsgStrPlural:
+		write = writePluralForms
+	default:
+		write = writeDefault
+	}
```

Comment:
We can rename `writePluralForms` to `writePlural` and `writeDefault` to `writeSingular` to use the same language and context.

Also, this makes it short and succinct.

```suggestion
	write := writeSingular

	if tag == MsgStrPlural {
		write = writePlural
	}
```